src/SkyApm.Diagnostics.Logging.Serilog/Sinks/SkywalkingSink.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Serilog.Core;
+using Serilog.Events;
+using Serilog.Formatting;
+using SkyApm.Tracing;
+using SkyApm.Transport;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace SkyApm.Diagnostics.Logging.Serilog.Sinks
+{
+    public class SkywalkingSink : ILogEventSink
```

Comment:
As @wu-sheng said, we try to avoid using skywalking in the code to avoid potential branding problems

Reply:
Yes, SkyApm is open to use.